src/docs/blog/2019-06-21-relaxing-react-jsx-no-bind-eslint-rule.md

```diff
@@ -4,7 +4,7 @@ authorURL: https://github.com/mrtnzlml
 title: Relaxing react/jsx-no-bind eslint rule
 ---
 
-Recently, I significantly relaxed Eslint rule [`react/jsx-no-bind`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md) (in [`@kiwicom/eslint-config`](https://www.npmjs.com/package/@kiwicom/eslint-config) version 4.10.0). This article tries to explain what was this rule for and why I decided to change it.
+Recently, I significantly relaxed Eslint rule [`react/jsx-no-bind`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md) (in [`@adeira/eslint-config`](https://www.npmjs.com/package/@kiwicom/eslint-config) version 4.10.0). This article tries to explain what was this rule for and why I decided to change it.
```

Comment:
`https://www.npmjs.com/package/@kiwicom/eslint-config` -> `https://www.npmjs.com/package/@adeira/eslint-config`?

Reply:
Please rebase - this file doesn't exist anymore if I remember correctly.

Reply:
Haha, I asked you to rebase my PR. 🙈 I must've been tired... 😬 

On it.

Reply:
> Haha, I asked you to rebase my PR,

That is what I figured as well 😆